### PR TITLE
Feature/rent service publisher

### PR DESCRIPTION
**Pull request by Antik88**

### CODE REVIEW

File: Rent.Service/Rent.Service.Application/Rents/Commands/CreateRentCommand.cs [hunk 1]
```diff
@@ -28,8 +41,28 @@ public async Task<RentModel> Handle(CreateRentCommand request, CancellationToken
...
+        var thingModel = await serviceRequest.GetFromServiceById<ThingModel>
+            (request.ThingId, _catalogUrl, cancellationToken);
+
+        var tenantModel = await serviceRequest.GetFromServiceById<UserModel>
+            (request.UserId, _userServiceUrl, cancellationToken);
+
+        var ownerModel = await serviceRequest.GetFromServiceById<UserModel>
+            (request.UserId, _userServiceUrl, cancellationToken);
+        //thingModel.OwnerId
+
```
Comment (5Ka-me) [inline Rent.Service/Rent.Service.Application/Rents/Commands/CreateRentCommand.cs]: Why is it here?

File: Rent.Service/Rent.Service.Application/Rents/Commands/CreateRentCommand.cs [hunk 2]
```diff
@@ -28,8 +41,28 @@ public async Task<RentModel> Handle(CreateRentCommand request, CancellationToken
...
+        if (_catalogUrl == null || _userServiceUrl == null)
+            throw new Exception();
+
+        var thingModel = await serviceRequest.GetFromServiceById<ThingModel>
+            (request.ThingId, _catalogUrl, cancellationToken);
+
+        var tenantModel = await serviceRequest.GetFromServiceById<UserModel>
+            (request.UserId, _userServiceUrl, cancellationToken);
+
+        var ownerModel = await serviceRequest.GetFromServiceById<UserModel>
```
Comment (5Ka-me) [inline Rent.Service/Rent.Service.Application/Rents/Commands/CreateRentCommand.cs]: What is the difference?

File: Rent.Service/Rent.Service.Application/Rents/Commands/CreateRentCommand.cs [hunk 3]
```diff
@@ -28,8 +41,28 @@ public async Task<RentModel> Handle(CreateRentCommand request, CancellationToken
             UserId = request.UserId,
         };
 
+        if (_catalogUrl == null || _userServiceUrl == null)
+            throw new Exception();
```
Comment (5Ka-me) [inline Rent.Service/Rent.Service.Application/Rents/Commands/CreateRentCommand.cs]: What exception?

**Review by 5Ka-me — CHANGES_REQUESTED** [review]

**Review by 5Ka-me — APPROVED** [review]

File: Rent.Service/Rent.Service.API/Program.cs [hunk 4]
```diff
@@ -24,6 +25,18 @@ public static void Main(string[] args)
...
 
+        builder.Services.AddMassTransit(configure =>
+        {
+            configure.UsingRabbitMq((ctx, cfg) =>
+            {
+                cfg.Host("localhost", "/", h =>
+                {
+                    h.Username("rmuser");
+                    h.Password("rmpassword");
+                });
```
Comment (Mmishaaa) [inline Rent.Service/Rent.Service.API/Program.cs:38]: u may store these values in appsettings

**Review by Mmishaaa — APPROVED** [review]

**Pull request merged**